profiles/xfce4.py

```diff
@@ -11,8 +11,7 @@ def _prep_function(*args, **kwargs):
 	for more input before any other installer steps start.
 	"""
 
-	# Gnome optionally supports xorg, we'll install it since it also
-	# includes graphic driver setups (this might change in the future)
+	# XFCE requires a functional xorg installation.
```

Comment:
Will need to keep an eye on the progress for wayland: https://wiki.xfce.org/releng/wayland_roadmap